scenarios/httpclient.benchmarks.yml

```diff
@@ -0,0 +1,228 @@
+﻿# Local run:
+# crank --config "C:\Users\knatalia\dev\git\Benchmarks\scenarios\httpclient.benchmarks.yml" --scenario httpclient-kestrel-get --profile local
+
+imports:
+  - https://github.com/aspnet/Benchmarks/blob/main/scenarios/aspnet.profiles.yml?raw=true
+  - https://raw.githubusercontent.com/dotnet/crank/main/src/Microsoft.Crank.Jobs.Wrk/wrk.yml
+
+variables:
+    warmup: 15 #seconds -- default value *copied* to WRK, see notes below
+    duration: 30 #seconds -- default value *copied* to WRK
+    serverPort: 1234 # -- default value *copied* to WRK
+    httpVersion: "1.1"
+    useHttps: false
+    numberOfHttpClients: 1
+    concurrencyPerHttpClient: 1 # unsupported by WRK -- will be ignored
+    collectRequestTimings: false # unsupported by WRK -- will be ignored
+
+    # HttpClient-specific parameters:
+    http11MaxConnectionsPerServer: 0 #unlimited
+    http20EnableMultipleConnections: true
+    useWinHttpHandler: false
+    useHttpMessageInvoker: false
+
+scenarios:
+  httpclient-kestrel-get:
+    server:
+      job: kestrel
```

Comment:
Can you try to use the new `agent: load` property (same level as job) so it binds to the `load` node in the existing profiles? This way we don't need a new profile. So when I start changing the other benchmarks too we can rename them with `primary/secondary/...` names for the machines in the profiles.